css/app.css

```diff
@@ -409,22 +388,30 @@ header.search-active #search-button {
   color: #B00;
 }
 
+header.search-active #search-counting {
+  display: block;
+}
+
 .search-navigation-button {
   border: 0;
-  border-left: 1px solid #DDD;
-  transition: all 200ms ease-in-out;
   background-position: 50% 50%;
   background-repeat: no-repeat;
   background-size: 21px;
   width: 0;
   height: 30px;
-  opacity: 0;
+  padding: 0;
```

Comment:
Re-arrange this block so that it's in alphabetical order.

Reply:
Done